pkg/kubelet/images/image_gc_manager.go

```diff
@@ -385,9 +390,9 @@ func (ev byLastUsedAndDetected) Less(i, j int) bool {
 	}
 }
 
-func isImageUsed(image container.Image, imagesInUse sets.String) bool {
+func isImageUsed(imageID string, imagesInUse sets.String) bool {
```

Comment:
Why make this change?

Reply:
Here we only need `imageID`. Also we only got imageID in `ImageRecords`.

Changing to `imageID` makes it more convenient for calling.